### ISSUE TITLE
Cannot work with joblib

### ISSUE THREAD

**Issue by WhateverLiu** [rstudio/reticulate#1288]
```
reticulate::repl_python()
from joblib import Parallel, delayed
def f(x): 
  return x + 1

rst = Parallel(n_jobs = 3)(delayed(f)(x) for x in range(9))
```
The code results in full error message: `OSError: [Errno 22] Invalid argument`.

**Comment by t-kalinowski** [comment]
Hi, I can't reproduce.
```python
> reticulate::repl_python()
Python 3.11.1 (/home/tomasz/.virtualenvs/r-reticulate/bin/python)
Reticulate 1.26.9000 REPL -- A Python interpreter in R.
Enter 'exit' or 'quit' to exit the REPL and return to R.
>>> %pip install joblib
Collecting joblib
Using cached joblib-1.2.0-py3-none-any.whl (297 kB)
Installing collected packages: joblib
Successfully installed joblib-1.2.0
>>> from joblib import Parallel, delayed
>>> def f(x): 
...     return x + 1
...     
... rst = Parallel(n_jobs = 3)(delayed(f)(x) for x in range(9))
>>> 
```

**Comment by t-kalinowski** [comment]
This was fixed with #1461 and/or #1458. 
Please try the development version of reticulate
```r
remotes::install_github("rstudio/reticulate")
```
